app/helpers/application_helper.rb

```diff
@@ -15,4 +15,24 @@ def like_or_dislike_btn(post)
       link_to('Like!', post_likes_path(post_id: post.id), method: :post)
     end
   end
+
+  def friendship_status(user)
+    if current_user.friend?(user)
+      content_tag(:span, 'Friend', class: 'badge badge-success')
+    elsif already_invited?(user.id)
+      content_tag(:span, 'Pending', class: 'badge badge-warning')
+    elsif current_user != user && !already_invited?(user.id)
+      link_to('Invite to friendship', "/users/#{user.id}/invite", class: 'badge badge-danger text-white')
+    end
+  end
 end
+
+# <% unless current_user == user %>
```

Comment:
- [ ] Please remove the commented out code before committing your changes.

Reply:
Resolved